waargonaut-deps.nix

```diff
@@ -1,92 +1,6 @@
 self: super: {
```

Comment:
Where is this file used?

Reply:
servant-waargonaut still uses it, and I was thinking I'd leave it there for a release or two with a deprecation warning (if such a thing is possible) and then drop it.

Probably too much of a faff for just a companion lib though >.>

Reply:
I wouldn't worry too much - it's not really worth it because it's so hard to make consistent package sets for downstream users.